JRC Publications Repository.js

```diff
@@ -0,0 +1,477 @@
+{
+	"translatorID": "03df2575-dbd9-49aa-9e58-1e6edd86d562",
+	"label": "JRC Publications Repository",
+	"creator": "Philipp Zumstein",
+	"target": "https?://(www\\.)?publications.jrc.ec.europa.eu/repository/(handle/|simple-search\\?|browse\\?)",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsbv",
+	"lastUpdated": "2014-12-27 11:22:03"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2014 Philipp Zumstein
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+
+var mappingTable = {
+	"Articles in Journals" : "journalArticle",
+	"Articles in books" : "bookSection",
+	"Books" : "book",
+	"eBook" : "book",
+	"Contributions to Conferences" : "conferencePaper",
+	"JRC Reference Reports" : "report",
+	"EUR - Scientific and Technical Research Reports" : "report",
+	"PhD Theses" : "thesis"
+}
+
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = ZU.xpath(doc, '//div[contains(@class, "discovery-result-results") or contains(@class, "panel-primary")]//table//a[contains(@href, "/repository/handle/")]');
+	for (var i=0; i<rows.length; i++) {
+		var href = rows[i].href;
+		var title = ZU.trimInternal(rows[i].textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+
+function detectWeb(doc, url) {
+	var type = ZU.xpath(doc, '//meta[contains(@name, "DC.type")]/@content');
+	if (type.length>0) {
+		if (mappingTable[type[0].textContent]) {
```

Comment:
If you want to use xpath to fetch the value of "content" attribute above, then you should use `.value` instead of `textContent` here. See https://developer.mozilla.org/en-US/docs/Web/API/Attr `.textContent` is deprecated on attributes